02week/ticTacToe.js

```diff
@@ -24,23 +26,65 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  if ((board[0][0] === 'X') && (board[0][1] === 'X') && (board[0][2] === 'X')) {
```

Comment:
You're repeating yourself A LOT here. You don't need to check for both X and O, just player turn

Reply:
Fixed.